docs/quickstart.rst

```diff
@@ -407,13 +437,15 @@ There is one other Datasheet that we need to configure for our model to run. The
     # Check the columns of the RunControl DataFrame
     >>> run_settings.info()
     <class 'pandas.core.frame.DataFrame'>
-    Index: 0 entries
-    Data columns (total 2 columns):
-     #   Column           Non-Null Count  Dtype 
-    ---  ------           --------------  ----- 
-     0   MinimumTimestep  0 non-null      object
-     1   MaximumTimestep  0 non-null      object
-    dtypes: object(2)
+    RangeIndex: 0 entries
+    Data columns (total 4 columns):
+    #   Column            Non-Null Count  Dtype
+    ---  ------            --------------  -----
+    0   MinimumTimestep   0 non-null      object
+    1   MaximumTimestep   0 non-null      object
+    2   MinimumIteration  0 non-null      object
```

Comment:
Maybe make a comment about how Iterations exist by default in the Run Control datasheet but are not used for this package